applications/asset_tracker/README.rst

```diff
@@ -114,8 +114,6 @@ For example, configure ``CONFIG_POWER_OPTIMIZATION_ENABLE`` to enable power opti
 In |SES|, select **Project** > **Configure nRF Connect SDK project** to browse and configure these options.
 Alternatively, use the command line tool ``menuconfig`` or configure the options directly in ``prj.conf``.
 
-.. include:: /includes/programming.txt
```

Comment:
Does this belong in this commit?

Reply:
Yes, because this is not needed anymore since SPM is programmed automatically.